corehq/apps/users/views/__init__.py

```diff
@@ -1270,3 +1270,9 @@ def register_fcm_device_token(request, domain, couch_user_id, device_token):
     user.fcm_device_token = device_token
     user.save()
     return HttpResponse()
+
+
+@memoized
+def has_custom_clean_password():
```

Comment:
nit: I would move this to the same module as the extension itself.

Reply:
sounds good, updated.